assets/css/editor.scss

```
@@ -45,6 +45,18 @@
 	}
 }
 
+svg.wc-block-editor-components-block-icon {
+	color: rgb(127, 84, 179);
```

One of the benefits of moving this to the CSS is that we can use the [CSS variable](https://github.com/Automattic/color-studio/blob/master/dist/color-variables.scss#L166), now: :slightly_smiling_face: 
```suggestion
	color: $studio-woocommerce-purple-50;
```

---

Nice! Thanks for pointing this out, @Aljullu. I should have checked that. 🙈